profiler/lib.profiler/src/org/netbeans/lib/profiler/results/BaseCallGraphBuilder.java

```diff
@@ -42,7 +42,7 @@
 
     //~ Instance fields ----------------------------------------------------------------------------------------------------------
 
-    protected List /*<Runnable>*/ afterBatchCommands = new ArrayList /*<Runnable>*/();
+    protected List <Runnable> afterBatchCommands = new ArrayList<>();
```

Comment:
See comment in `CCTResultsFilter.java`.

Reply:
I should have thought about that.. But, when I start making these mass changes, I kinda get into a rhythm.

And that's why we do code reviews..

Thanks for the code review!